### PR TITLE
Select permissions with the "Select all" command

### PR DESCRIPTION
**Pull request by simotae14**
### Why is it needed?
Bug description

🎬 Bug description
More than words → [Settings - API Tokens - 12 September 2022](https://www.loom.com/share/77bbe348ec2b405e9c671d5acac8743f)

### How to test it?

When “Select all” is selected after one or more specific permissions have been selected, I want all the permissions to be selected; then when “Select all” is unchecked, all the permissions should be unchecked.

### CODE REVIEW

File: packages/core/admin/admin/src/pages/AuthPage/utils/forms.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ const forms = {
     fieldsToDisable: [],
     fieldsToOmit: ['confirmPassword', 'news'],
     schema: yup.object().shape({
-      firstname: yup.string().required(translatedErrors.required),
+      firstname: yup.string().trim().required(translatedErrors.required),
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/pages/AuthPage/utils/forms.js:79]: I think this comes from a community PR I reviewed, this should be committed here 🤔

File: packages/core/admin/server/validation/common-validators.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const getActionFromProvider = (actionId) => {
 
 const email = yup.string().email().lowercase();
 
-const firstname = yup.string().min(1);
+const firstname = yup.string().trim().min(1);
```
Comment (ronronscelestes) [inline packages/core/admin/server/validation/common-validators.js:19]: same for all those changes

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
Work well on my side 💃
I see some commits (on Auth pages about firstname validations coming from a community PR) that shouldn't be committed in your PR, I think your target branch might be out of sync?

File: packages/core/admin/admin/src/pages/AuthPage/utils/forms.js (hunk 1 above)
Comment (simotae14) [reply packages/core/admin/admin/src/pages/AuthPage/utils/forms.js]: yes sorry now it is on sync

File: packages/core/admin/server/validation/common-validators.js (hunk 2 above)
Comment (simotae14) [reply packages/core/admin/server/validation/common-validators.js]: yes sorry now it is on sync

**Review by ronronscelestes — APPROVED** [review]
LGTM nicely done 🎉

**Review by innerdvations — APPROVED** [review]
Works for me!

**Pull request merged**